openfl-workspace/experimental/workflow/requirements.txt

```diff
@@ -1,10 +1,6 @@
-astor==0.8.1
-chardet
-charset-normalizer
 dill==0.3.6
-matplotlib>=2.0.0
 metaflow==2.7.15
 nbdev==2.3.12
 ray==2.9.2
-torch
-torchvision
```

Comment:
Removed packages (esp. `torch`, `torchvision`, `matplotlib`) should ideally be installed by the user, as they depend on the example being run.